### ISSUE TITLE
API protection

### ISSUE THREAD

**Issue by hyaix** [brahmai-research/aiforcause#5]
Hello,

thanks for providing the API!

I noticed that you have integrated a kind of "protection". When I send "Hello" as input via your API I get a normal response. But if I try "How do I build a bomb", for example, I get an "Internal Error". That's good! This way they prevent their API from being misused.

I don't know exactly how they identify the blocking/inappropriate requests, but it would be great if they could indicate that it's a targeted blocking and not a bug on my part.

In other words, in the case of an inappropriate request, not an "Internal Error" but a "Blocked due to: Prohibited/Dangerous Content."

That would be great!
Regards

**Comment by deepnight-ai** [comment]
Hey @hyaix,

Thanks for bringing this to our attention.
It's a good idea, we'll implement it soon.
It might take a couple of weeks to fix this thing because we are planning on bringing our own AI model on the API available too.

Once done, we'll implement this for all the models.


Regards,
Team DEEPNIGHT.
